odoo/addons/base/models/ir_qweb.py

```diff
@@ -1182,11 +1242,9 @@ def _compile_node(self, el, compile_context, level):
             return self._compile_static_node(el, compile_context, level)
 
         path = compile_context['root'].getpath(el)
-        if compile_context['last_path_node'] != path:
-            compile_context['last_path_node'] = path
-            body = [indent_code(f'log["last_path_node"] = {path!r}', level)]
-        else:
-            body = []
+        xml = etree.tostring(etree.Element(el.tag, el.attrib), encoding='unicode')
```

Comment:
for the record, this extra step could take about 6 microseconds per node during template computation, according to this non-scientific micro-benchmark:

```py
In [4]: doc = etree.fromstring("<a f='1' b='2' c='3' d='4' e='5'/>")
In [5]: %timeit etree.tostring(etree.Element(doc.tag, doc.attrib), encoding='unicode')
6.1 µs ± 60.8 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)
```